services/src/main/java/org/keycloak/authentication/authenticators/conditional/ConditionalLoaAuthenticator.java

```diff
@@ -0,0 +1,75 @@
+package org.keycloak.authentication.authenticators.conditional;
```

Comment:
It might be good if you could add header comments the same as other newly added source files like AuthenticatorUtil.java .